widgetssdk/src/main/java/com/glia/widgets/core/secureconversations/domain/HasOngoingInteractionUseCase.kt

```diff
@@ -0,0 +1,52 @@
+package com.glia.widgets.core.secureconversations.domain
+
+import com.glia.widgets.chat.domain.IsAuthenticatedUseCase
+import com.glia.widgets.core.secureconversations.SecureConversationsRepository
+import com.glia.widgets.di.GliaCore
+import com.glia.widgets.entrywidget.EntryWidget
+import com.glia.widgets.helper.unSafeSubscribe
+import com.glia.widgets.launcher.EngagementLauncher
+import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers
+import io.reactivex.rxjava3.core.Observable
+import java.util.concurrent.TimeUnit
+
+internal class HasOngoingInteractionUseCase(
```

Comment:
Does it make sense to add something that will say that this use case is related to Secure Conversations?

Reply:
renamed